src/components/Pagination.svelte

```diff
@@ -101,22 +106,22 @@
 {#if itemsPerPage < totalItems}
   <div class="pagination-bar">
     <div
-      on:click|preventDefault={() => changePage($currentPage !== 1 ? $currentPage - 1 : 1)}
-      class="pagination-button {$currentPage === 1 ? 'current-page' : ''}">
+      on:click|preventDefault={() => changePage(currentPage !== 1 ? currentPage - 1 : 1)}
+      class="pagination-button {currentPage == 1 ? 'current-page' : ''}">
```

Comment:
I wonder why == is giving the correct result instead of ===, as is the svelte syntax. I observed this by accident.

Reply:
Oh okay, then it must be due to different data types. I'll update this.

Reply:
Ah that makes sense, yes. You might need to cast the value to integer before doing the comparison